### PR TITLE
Added account page

### PR DESCRIPTION
**Pull request by TwiN**
This PR adds a simple account page that gets the data from the bot's API.

account page

Obviously, this is a simple implementation, but it could help reduce requests by minimizing the use of the `!balance` commands.

For now, I didn't actually create a separate HTML page because I felt it was unnecessary.

I also fixed a few minor derps (e.g. logo not centered,  `bugs` link not opening in new tab, etc.)

Let me know what you think.

-----------

**UPDATE:** Bot went down the moment I created this PR, so testing it will have to wait until it's up again.

### CODE REVIEW

File: docs/css/style.css [hunk 1]
```diff
@@ -12,12 +12,13 @@ nav{
   width:auto;
   vertical-align:middle;
   margin-right:5px;
-  padding-top:0px;
+  padding-top: 0;
+  position: relative;
```
Comment (TwinProduction) [inline docs/css/style.css]: centers logo

**Comment by thecsw** [comment]
OOOh, very nice! If everything is ok, some dev will merge this. Thanks!

**Comment by thecsw** [comment]
Do you have some github.io page or something where I can take a look?

P.S. We have some merge conflicts! Please resolve them and send a testing webpage.

**Comment by TwiN** [comment]
That was one hell of a conflict. My head hurts.

**Comment by TwiN** [comment]
@thecsw you can just open `index.html` with a browser. Since it uses https://memes.market/api, it should work even without the bot running on your local machine.

But basically, all I did for the account 'page' is adding a container at the end of `index.html` that is hidden unless the "account" navigation button is pressed, in which case the home container is hidden and the account container is shown.

Other than that, I've refactored `index.html` to fix inconsistent indentation.

--------

**EDIT:** @thecsw created a quick testing page https://twinproduction.github.io/ghpagetest/

**Comment by TwinProduction** [comment]
@thecsw created a quick testing page https://twinproduction.github.io/ghpagetest/

**Comment by robalb** [comment]
there was already a search bar in the nav bar, and since the data to show in the user page is very small, i just moved it in a simpler modal box

**Comment by TwiN** [comment]
@robalb Good idea, hadn't thought of that until I saw the modal that was added for the upcoming economy reset when I resolved the conflicts.

I was also working on that on my end, but you beat me to it.

**Comment by robalb** [comment]
ok, i think the account page is ready. In future we'll probably need a separate page to show more data, (like all the active investments with a graph) and have te ability to link directly to it from the reddit bot, so that users don't have to always type in their username

**Comment by TwiN** [comment]
@robalb I was thinking of creating another page too, but then I thought about all the redundant code we'd have to deal with (e.g. navbar, scripts, etc.).

Though, we could probably add a parameter (e.g. `index.html?account=twinproduction`) and when the page is finished loading, check if there's a parameter `account` in the url, if there's one, then display the account information.

-----

UPDATE: I'm working on that

**Comment by robalb** [comment]
@TwinProduction yeah, but it's probably not worth it. We are planning to integrate a reddit client in the website, to view all the posts in memeconomy in real time. when we'll do it, there will be enough stuff to justify a separate user page with a direct url

**Comment by TwiN** [comment]
@robalb no worries, for now, I just added a very tiny snippet to do that:

```js
let url = new URL(window.location.href);
let username = url.searchParams.get("account");
if (username) {
    document.getElementById('investor-username').value = username;
    showAccount('investor-username');
    history.pushState(null, '', window.location.href.split('?')[0]);
}
```

It automatically opens up the account modal if the url contains `?account=...` and removes the `account` parameter from the url afterward. 

Live example: https://twinproduction.github.io/ghpagetest/?account=twinproduction

**Comment by robalb** [comment]
ok, i just cleared a bit the code moving everything related with the user modal into a closure module

**Comment by TwiN** [comment]
Looks good to me.
Push when you're ready @robalb @thecsw 

There's quite a few changes in this PR, so it's probably better to push it sooner than later or we'll end up with a long list of conflicts.

**Comment by robalb** [comment]
uhmm i'm an idiot
`?account=</img>`

**Comment by TwiN** [comment]
@robalb did you introduce that xss, or was that me?

I don't remember directly parsing the account parameter o.O

-------

**EDIT:** It's because of the `${username} is not an investor` message that you added. ~~Can you fix it or should I?~~ see https://github.com/MemeInvestor/memeinvestor_bot/pull/78

**Pull request merged**